src/main/java/types/Cluster.java

```diff
@@ -522,4 +522,22 @@
      * @since 4.4.5
      */
     FipsMode fipsMode();
+
+    /**
+     * Indicates if an upgrade has been started for the cluster.
+     *
+     * @author Scott Dickerson <sdickers@redhat.com>
+     * @date 17 Mar 2022
+     * @since 4.5.0
+     */
+    Boolean upgradeInProgress();
```

Comment:
A `upgradeCorrelationId == null` could indicate no upgrade.

Reply:
This field just surfaces a field already on the DB table.